api/v1alpha1/packagebundle_types.go

```diff
@@ -113,14 +113,15 @@ type PackageBundleStatus struct {
 	State PackageBundleStateEnum `json:"state"`
 }
 
-//+kubebuilder:validation:Enum={"inactive","active","active (upgrade available)","ignored"}
+//+kubebuilder:validation:Enum={"inactive","active","active (upgrade available)","ignored","ignored version"}
 type PackageBundleStateEnum string
 
 const (
 	PackageBundleStateInactive         PackageBundleStateEnum = "inactive"
 	PackageBundleStateActive           PackageBundleStateEnum = "active"
 	PackageBundleStateUpgradeAvailable PackageBundleStateEnum = "active (upgrade available)"
 	PackageBundleStateIgnored          PackageBundleStateEnum = "ignored"
```

Comment:
I find "ignored" and "ignored version" not mutually exclusive by description. Is there a better name convention that can help address this issue?

Reply:
Yeh, I'm not a big fan of the name either. I don't like state names with spaces in there. I don't like "active (upgrade available)", but that is where we went for for whatever reason. I'd kind of sooner PackageBundleStateUpgradeAvailable didn't exist just PackageBundleStateActive with a NOTE "upgrade available"

Reply:
I hear what you are saying. Not for this PR, but I think if we are having doubt about those names, we should change it prior to GA. It seems to be better to address it early on before we accumulate a bigger user base.